migrations/versions/cfd041b17636_.py

```diff
@@ -0,0 +1,76 @@
+"""empty message
+
+Revision ID: cfd041b17636
+Revises: 2ef73ff71f0c
+Create Date: 2021-12-21 00:36:01.247520
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'cfd041b17636'
+down_revision = '2ef73ff71f0c'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+import app
+import app.extensions
+
+
+
+def upgrade():
+    """
+    Upgrade Semantic Description:
+        ENTER DESCRIPTION HERE
+    """
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('relationship',
+    sa.Column('created', sa.DateTime(), nullable=False),
+    sa.Column('updated', sa.DateTime(), nullable=False),
+    sa.Column('guid', app.extensions.GUID(), nullable=False),
+    sa.Column('start_date', sa.DateTime(), nullable=True),
+    sa.Column('end_date', sa.DateTime(), nullable=True),
+    sa.Column('type', sa.String(), nullable=True),
```

Comment:
is `type` able to be None/null?  just curious - have no info myself about it.

Reply:
I would venture that it should, especially since migration will not have a type for existing ones.